plugin.video.tyt/resources/language/English/strings.po

```diff
@@ -1,15 +1,22 @@
 msgid ""
 msgstr ""
 
-msgctxt "#3000"
+msgctxt "#30000"
```

Comment:
i don't claim to understand the difference (and it would appreciated if someone could enlighten me), but plugins should use the 30000-31000 range :
https://github.com/xbmc/xbmc/blob/master/addons/resource.language.en_gb/resources/strings.po#L15292